src/core/lib/security/credentials/ssl/ssl_credentials.h

```diff
@@ -69,7 +69,16 @@ class grpc_ssl_credentials : public grpc_channel_credentials {
                     grpc_ssl_pem_key_cert_pair* pem_key_cert_pair,
                     const grpc_ssl_verify_peer_options* verify_options);
 
+  grpc_security_status InitializeClientHandshakerFactory(
```

Comment:
Can we add a comment describing this function?

Reply:
Friendly ping :)

Reply:
PTAL